Makefile

```diff
@@ -139,6 +141,12 @@ deps: .state/env/pyvenv.cfg
 	rm -r $(TMPDIR)
 	$(BINDIR)/pip check
 
+github-actions-deps:
+ifneq ($(GITHUB_HEAD_REF), false)
+	git fetch origin $(GITHUB_HEAD_REF):refs/remotes/origin/$(GITHUB_HEAD_REF)
+	git diff --name-only $(GITHUB_HEAD_REF) | grep '^requirements/' || exit 0 && $(MAKE) deps
+endif
+
```

Comment:
Something's broken with this: https://github.com/pypa/warehouse/pull/9083/checks?check_run_id=1885648471

```
Run make github-actions-deps BINDIR="$(dirname $(which python))"
/bin/sh: 1: .state/env/bin/python: not found
git fetch origin github-actions-ci:refs/remotes/origin/github-actions-ci
From https://github.com/pypa/warehouse
 * [new branch]        github-actions-ci -> origin/github-actions-ci
git diff --name-only github-actions-ci | grep '^requirements/' || exit 0 && make deps
fatal: ambiguous argument 'github-actions-ci': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

Reply:
Fixed!